### PR TITLE
Auqa 85 popup verification

### PR DESCRIPTION
**Pull request by abhijitvigs**

### CODE REVIEW

File: Testcases/UITests/uiPopupTest.robot [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Resource    ${EXECDIR}/Resources/UIResources/uiPopupResources.robot
+Resource    ${EXECDIR}/Inputs/testInputs.robot
+
+
+*** Test Cases ***
+VXPopupMessages
+    [Setup]    popupResources.popupTestSetup
+    popupResources.suppressAlarmPopup
+
+CXPopupMessages
```
Comment (ananiac) [inline Testcases/UITests/uiPopupTest.robot:17]: Comment this for now

File: Testcases/UITests/uiPopupTest.robot [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    log to console  override
```
Comment (ananiac) [inline Testcases/UITests/uiPopupTest.robot:18]: update log to console   CXPopupMessages yet to be implemented  and comment it or remove it

File: ExternalKeywords/readExcel.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 auqa_dir= os.path.dirname(os.path.abspath('/home/fc/automation/AuQA/execution.py'))
```
Comment (ananiac) [inline ExternalKeywords/readExcel.py:4]: This file  ExternalKeywords/readExcel.py-should be as in masters

File: PageObjects/vxAIEngine.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# VX AIEngine - Site Editor Web Elements
+
+# VX AIEngine - Notifications tab
+# notifications_tab="id=tab-1306-btnInnerEl"
```
[the remark this answers is not in the document]
Comment (ananiac) [reply PageObjects/vxAIEngine.py:4]: Inputs/testInputs.xlsx has conflicts - still has conflicts

[the remark this answers is not in the document]
Comment (ananiac) [reply PageObjects/vxAIEngine.py:4]: Rename - vxAIEngine.py

File: PageObjects/vxAIEngine.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+# notifications_tab="xpath=//span[@id='tab-1306-btnInnerEl']"
```
Comment (ananiac) [inline PageObjects/vxAIEngine.py:5]: remove unwanted comments and unwanted commented lines

File: PageObjects/vxAIEngine.py [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+# notifications_tab="xpath=//span[contains(text(),'Notifications')]"
+notifications_tab="xpath=//span[contains(text(),'Notifications')]"
+
+suppress_alarm_button="xpath=//span[contains(text(),'Suppress Alarm')]"
+suppress_alarm_ok_button="xpath=//span[contains(text(),'OK')]"
+suppress_alarm_title="""xpath=//*[@id="messagebox-1001_header-title-textEl"]"""
+suppress_alarm_popup_message="""xpath=//*[@id="messagebox-1001-msg"]"""
+selected_suppress_alarm_popup_message="""xpath=//*[@id="messagebox-1001-msg"]"""
+
+#suppress_alarm_no_button="xpath=//span[contains(text(),'No')]"
```
Comment (ananiac) [inline PageObjects/vxAIEngine.py:17]: Remove the commented line

File: PageObjects/vxAIEngine.py [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+suppress_alarm_no_button="xpath=//span[@id='button-1007-btnEl']"
+
+suppressed_alarms_button="xpath=//span[contains(text(),'Suppressed Alarms')]"
+suppressed_alarm_ok_button="xpath=//span[contains(text(),'OK')]"
+suppressed_alarms_title="""xpath=//*[@id="window-1404_header-title-textEl"]"""
+unsuppress_button="xpath=//span[contains(text(),'Unsuppress')]"
+restore_suppressed_alarm_title="""//*[@id="messagebox-1001_header-title-textEl"]"""
+unsuppress_all_button="xpath=//span[contains(text(),'Unsuppress All')]"
+restore_all_suppress_alarm_button="""xpath=//*[@id="button-1007-btnEl"]"""
+#restore_all_suppress_alarm_button="xpath=//span[contains(text(),'No')]"
```
Comment (ananiac) [inline PageObjects/vxAIEngine.py:27]: Remove the commented line

File: Resources/UIResources/uiPopupResources.robot [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+
+suppressedAlarmPopup
+    startBrowserAndLoginToAIEngineVX
+    set selenium timeout  ${short_wait_time}
+    sleep  ${load_time}
+    click element   ${notifications_tab}
+    log to console  Notifications tab clicked
+    set selenium timeout  ${short_wait_time}
+    sleep  ${load_time}
+
```
Comment (ananiac) [inline Resources/UIResources/uiPopupResources.robot:75]: Remove the space

File: Resources/UIResources/uiPopupResources.robot [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+    click element  ${suppressed_alarms_button}
+    log to console  'Suppressed Alarms' button clicked
+
+    # verify 'Suppressed Alarms' popup title
+    ${verify_suppressed_alarms_popup_title}=  get text  ${suppressed_alarms_title}
+    should be equal	 ${verify_suppressed_alarms_popup_title}  ${test_input}[suppressed_alarm_popup_title]
+    log to console  Popup title is "${verify_suppressed_alarms_popup_title}"
+    set selenium timeout  ${short_wait_time}
+    sleep  ${load_time}
+
```
Comment (ananiac) [inline Resources/UIResources/uiPopupResources.robot:86]: Remove line space

File: Resources/UIResources/uiPopupResources.robot [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+    # 'Unsuppressed' button
+    click element  ${unsuppress_button}
+    log to console  'Unsuppressed' button clicked
+
+    verifyTitleOfPopUp  ${restore_suppressed_alarm_title}  ${test_input}[restore_all_suppressed_alarms_title]
```
Comment (ananiac) [inline Resources/UIResources/uiPopupResources.robot:91]: remove commented lines

File: Resources/UIResources/uiPopupResources.robot [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+    should be equal	 ${popup_message}  ${test_input}[suppress_alarm_popup_message]
+    log to console  Popup message is "${popup_message}"
+    click element  ${suppressed_alarm_ok_button}
+    sleep  ${load_time}
+
+    # 'Unsuppress All' button
+    click element  ${unsuppress_all_button}
+    log to console  'Unsuppress All' button clicked
+
+#    # verify 'Restore All Suppressed Alarms' popup title
```
Comment (ananiac) [inline Resources/UIResources/uiPopupResources.robot:110]: Remove the commented block

File: Resources/UIResources/uiPopupResources.robot [hunk 12]
```diff
@@ -0,0 +1,185 @@
...
+#    sleep  ${load_time}
+
+verifyTitleOfPopUp
+    [Arguments]    ${restore_suppressed_alarm_title}  ${test_input}[restore_all_suppressed_alarms_title]
+    ${verify_popup_title}=  get text  ${restore_suppressed_alarm_title}
+    should be equal	 ${verify_popup_title}  ${test_input}[restore_suppressed_alarm_title]
+    log to console  Popup title is "${verify_popup_title}"
+    sleep  ${load_time}
+
+#    ${verify_restore_suppressed_alarm_title}=  get text  ${restore_suppressed_alarm_title}
```
Comment (ananiac) [inline Resources/UIResources/uiPopupResources.robot:132]: Remove the commented block

File: Testcases/UITests/uiPopupTest.robot [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+
+*** Test Cases ***
+VXPopupMessagesVerification
+    [Setup]    uiPopupResources.popupTestSetup
+    #uiPopupResources.suppressAlarmPopup
+    uiPopupResources.suppressedAlarmPopup
+#    uiPopupResources.overridesPopup
+#    uiPopupResources.claerOverridePopup
+
+#CXPopupMessages
```
Comment (ananiac) [inline Testcases/UITests/uiPopupTest.robot:20]: Remove commented block

File: Testcases/UITests/uiPopupTest.robot [hunk 14]
```diff
@@ -0,0 +1,23 @@
...
+Resource    ${EXECDIR}/Resources/UIResources/uiPopupResources.robot
+Resource    ${EXECDIR}/Inputs/testInputs.robot
+
+
+*** Test Cases ***
+VXPopupMessagesVerification
+    [Setup]    uiPopupResources.popupTestSetup
+    #uiPopupResources.suppressAlarmPopup
+    uiPopupResources.suppressedAlarmPopup
+#    uiPopupResources.overridesPopup
```
Comment (ananiac) [inline Testcases/UITests/uiPopupTest.robot:17]: Remove the commented lines

**Comment by abhijitvigs** [comment]
New branch AUQA-85_uiPopupVerification is created

**Pull request closed without merging**